geomet_data_registry/util.py

```diff
@@ -26,6 +26,47 @@
 DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 
 
+class VRTDataset:
+    """
+    Object used to create VRTs. The object can be passed a list of ordered
+    variable names to sort the provided file paths on and create a VRT with the
+    bands in the specified list order (ex. UU layers where the U data needs
+    to be the first band and the V data the second band).
+    """
+    def __init__(self, filepaths, raster_x_size=None, raster_y_size=None,
+                 bands_order=None):
+        self.vrt_dataset = [
+            '<VRTDataset rasterXSize="{}" rasterYSize="{}" bands="{}">'.format(
+                raster_x_size, raster_y_size, len(filepaths)),
+            '</VRTDataset>'
+        ]
+        self.vrt_raster_band_template = '''
+<VRTRasterBand dataType="Byte" band="{}">
+    <ComplexSource>
+        <SourceFilename>{}</SourceFilename>
+        <SourceBand>1</SourceBand>
+        <ScaleOffset>0.0</ScaleOffset>
+        <ScaleRatio>1.0</ScaleRatio>
+    </ComplexSource>
+</VRTRasterBand>
+        '''.replace('\n', '')
```

Comment:
are you collapsing the string here into one line?  Perhaps add a specific class function instead?

Reply:
Added a `VRTDatateset.formatted_vrt()` method to format the VRT appropriately (single line with no whitespace formatting).